Jenkinsfile

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env groovy
 def artifactoryServer = Artifactory.server('Digital-Artifactory')
 
-@Library(['PPCmanifest','security-ci-commons-shared-lib']) _
-def NODE = nodeDetails("uaa-upgrade")
+@Library(['PPCmanifest']) _
```

Comment:
This library is not used anymore; let's remove. It was used in the past to publish build artifacts to BinTray for onprem Rosneft customer. The stage to publish to BinTray has been removed, so no need for the library.

Reply:
Done, removed PPCmanifest lib